modules/loader-utils/src/lib/node/stream.browser.ts

```diff
@@ -1 +1,3 @@
+export const Transform = null;
```

Comment:
I am not sure Transform is still needed. I think it was used in the Parquet module but I thought I commented it out.